lib/ord.js

```diff
@@ -40,32 +41,38 @@ const initializeAppConfig = (csn) => {
     for (const key of modelKeys) {
         const keyDefinition = csn.definitions[key];
         switch (keyDefinition.kind) {
-            case ORD_RESOURCE_TYPE.service:
+            case CDS_ELEMENT_KIND.service:
                 if (!keyDefinition["@cds.external"]) {
                     serviceNames.push(key);
                 }
                 break;
             // TODO: should be rewritten
-            case ORD_RESOURCE_TYPE.entity:
-                if (!key.includes(".texts") && keyDefinition["@ODM.entityName"]) {
-                    odmEntity.push(createEntityTypeTemplate(keyDefinition));
+            case CDS_ELEMENT_KIND.entity:
+                if (!key.includes(".texts")) {
+                    endpoints.add(key);
+                    if (keyDefinition["@ODM.entityName"]) {
+                        odmEntities.push(createEntityTypeMappingsItemTemplate(keyDefinition));
+                    }
                 }
                 break;
-            case ORD_RESOURCE_TYPE.event:
+            case CDS_ELEMENT_KIND.event:
                 events.push(key);
                 break;
+            case CDS_ELEMENT_KIND.action:
+            case CDS_ELEMENT_KIND.function:
+                endpoints.add(key);
+                break;
         }
     }
 
-    odmEntity = _.uniqBy(odmEntity, "ordId");
-
     return {
         env: cds.env["ord"],
         lastUpdate,
         appName,
+        endpoints: Array.from(endpoints),
```

Comment:
what is the meaning of endpoints? I would suggest provide more context in the name

Reply:
In the cds service files, entities, actions and functions will result into API endpoints. That's why I gave them the common name - "endpoints". On the other side, no endpoints => no apiResource.

Reply:
thx, understand after your explanation, then could you rename it as apiEndpoints?